### ISSUE TITLE
(Feature Request): Alternative rate bar color scheme (suggested by MHX)

### ISSUE THREAD

**Issue by mphelp** [Anarios/return-youtube-dislike#186]
### Extension or Userscript?

Both

### Request or suggest a new feature!

From MHX in discord: Add an option in the extension menu to make the like/dislike bar either Gray (Default) or Green / Red

![image](https://user-images.githubusercontent.com/25730130/144712068-118d8b8d-a20c-41bf-9e7d-9e2e667e3781.png)

### Ways to implement this!

We already have a settings popup. We should add a toggle which says something along the lines of "Green / Red rating bar." Note: This is low priority and I'm happy to work on this, but probably not today.

### Can you work on this?

- [X] Yes
- [ ] No

**Comment by SirLich** [comment]
If you do this, I would highly suggest making this light/dark theme dependent, and also use different colors. Those are very aggresive. 

I also personally don't see anything wrong with YTs blue line.

**Comment by mphelp** [comment]
@SirLich I agree on the color harshness. And around the default colors, I do think they're fine, but it doesn't hurt to provide users with options
